packages/medusa/src/api/routes/admin/orders/index.ts

```diff
@@ -1,20 +1,36 @@
 import { Router } from "express"
 import "reflect-metadata"
 import { Order } from "../../../.."
-import { DeleteResponse, PaginatedResponse } from "../../../../types/common"
-import middlewares from "../../../middlewares"
+import {
+  DeleteResponse,
+  EmptyQueryParams,
+  PaginatedResponse,
+} from "../../../../types/common"
+import middlewares, { transformQuery } from "../../../middlewares"
+import { AdminGetOrdersParams } from "./list-orders"
+import { FlagRouter } from "../../../../utils/flag-router"
 
 const route = Router()
 
-export default (app) => {
+export default (app, featureFlagRouter: FlagRouter) => {
   app.use("/orders", route)
 
+  let relations = [...defaultAdminOrdersRelations]
+  if (featureFlagRouter.isFeatureEnabled("sales_channels")) {
+    relations = [...relations, "sales_channel"]
```

Comment:
suggestion: Here we can just push the result without spreading. what do you think?